app/controllers/general.php

```diff
@@ -16,12 +16,23 @@
 use Appwrite\Network\Validator\Origin;
 use Appwrite\Storage\Device\Local;
 use Appwrite\Storage\Storage;
+use Appwrite\Utopia\Response\Filter;
+use Appwrite\Utopia\Response\Filter\V06;
 use Utopia\CLI\Console;
 
 Config::setParam('domainVerification', false);
 Config::setParam('cookieDomain', 'localhost');
 Config::setParam('cookieSamesite', Response::COOKIE_SAMESITE_NONE);
 
+function mapResponseFormatToClass(string $responseFormat): Filter {
+    switch($responseFormat) {
+        case preg_match($responseFormat, "/0\.[0-6]\.\d?/"):
+            return new V06();
```

Comment:
Does this cause a new instance of the filter to be created upon each request? 
If yes, I should probably implement the Filter class as a Singleton. 
WDYT @eldadfux

Reply:
see my second comment about this.